src/main/resources/META-INF/reaktivity/core.idl

```diff
@@ -81,6 +107,7 @@ scope core
 
         struct Data extends core::stream::Frame [0x00000002]
         {
+            int64 authMask; // TODO: default 0L
```

Comment:
I think we should remove this until we know it's needed and what purpose it will serve (it is ambiguous, could referring to the roles of the sender or the required roles for the recipient)